### ISSUE TITLE
Completion widget not keyboard navigable

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#173626]
Ref #173622

Assuming we could fix the ghost text issue and give screen readers users a hint how to open the widget—if that's the right thing to do—it looks like we could then continue on to ensure that the tooltip is reading more than just the current suggestion.

From what I can tell, it isn't possible to use the keyboard to navigate through the widget actions. Is there a way to arrow through these without dismissing it? Could we put focus on the first enabled button when it pops up? There may be other ways. Just getting the conversation started.

https://user-images.githubusercontent.com/25163139/217103076-924e2dbc-95b1-4529-830a-5b527155f854.mp4

cc @isidorn @hediet @esonnino @meganrogge

**Comment by isidorn** [comment]
Good point. Keyboard navigation should just work.
@daviddossett if you use turn on accessibility mode, Cmd+K, Cmd+I to show the hover - does moving focus still not work via keyboard?

**Comment by meganrogge** [comment]
I can reproduce this. I think it's because usually tab is what is used for navigation through the elements, but in this case, it accepts the suggestion

**Comment by daviddossett** [comment]
I'm wondering what the best alternative would be seeing as tab is the way to accept the suggestion? Obviously we don't want to break that behavior but I can see how it would be frustrating for users who are used to hitting tab to focus.

**Comment by meganrogge** [comment]
oh pressing `Alt` fixes it, so just hold `Alt` to navigate

**Comment by daviddossett** [comment]
For tab+enter accept you mean? I think that makes sense given that we can show the hint in the hover itself.

> oh pressing Alt fixes it, so just hold Alt to navigate

Is that a typical interaction in this kind of situation? Would be curious if screen reader users would know to do that vs. just hitting tab and accidentally accepting.

**Comment by meganrogge** [comment]
@Tyriar would know but looks like `Alt` is how one locks the hover

**Comment by meganrogge** [comment]
I was suggesting that we'd update the tooltip in the hover to say press `Tab+Enter` to accept instead of `Tab` here. 

But I actually think that pressing `Alt` is what is expected here and then tabbing works besides the bug above.

![Screenshot 2023-02-07 at 11 54 48 AM](https://user-images.githubusercontent.com/29464607/217326889-71197380-078e-4a32-9c01-c1420499d37e.png)

**Comment by Tyriar** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/130562?

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22feature-request%22%20label%3A%22accessibility%22%20label%3A%22inline-completions%22). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
